home-dir/homeDir.js

```diff
@@ -1,8 +1,7 @@
+import os from 'os';
```

Comment:
`os` is a big library and if only one function `homedir` is being used, then only that function should have been imported.